rust-toolchain.toml

```diff
@@ -18,5 +18,8 @@
 # Toolchain override for rustup
 
 [toolchain]
-channel = "nightly-2023-12-18"
+channel = "nightly-2024-05-15"
+components = [ "rust-src" ]
```

Comment:
I suppose this won't add too much size increase for `no-std` CI, right?

Reply:
Building `std` needs the components `rust-src` and it increase the download size indeed, but setting the profile as minimal removes the components we do not need (at line 24-25) which reduces the download size:
```
# minimal profile: install rustc, cargo, and rust-std
profile = "minimal"
```

By default it will download `cargo`, `clippy`, `rust-docs`, `rustc`, `rustfmt`...

Reply:
Okay that's fine as long as the CI is happy.